packages/bpk-component-accordion/src/BpkAccordionItem.js

```diff
@@ -45,14 +45,13 @@ const BpkAccordionItem = props => {
     );
   }
 
-  const titleId = `${id}_title`;
   const contentId = `${id}_content`;
 
   return (
     <div id={id} {...rest}>
       <dt
-        role="heading"
```

Comment:
ARIA role must be appropriate for the element
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role
This Element type cannot support the Role `Heading`

Reply:
In my mind, it does seem appropriate for it to be a heading though...  Maybe the `dt` tag should be replaced with `h3`? wdyt?

Reply:
We could set a child element of the `<dt/>` to have the role set on it but as it stands the dt tag roles are limited to Dialog roles only https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role and maybe some others but not heading unfortunately.

Reply:
Sorry not dialog - definition roles see.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute Example 6